l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -122,6 +134,19 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
         eraFirstPostUpgradeBatch = _eraFirstPostUpgradeBatch;
     }
 
+    /// @dev This sets the first post upgrade batch for era, used to check old withdrawals
+    /// @param  _eraLastOldBridgeDepositBatch The the zkSync Era batch number that processes the last deposit tx initiated by the legacy bridge
+    /// @param _eraLastOldBridgeDepositTxNumber The tx number in the _eraLastOldBridgeDepositBatch of the last deposit tx initiated by the legacy bridge
+    function eraLastOldBridgeDepositDeadline(uint256 _eraLastOldBridgeDepositBatch, uint256 _eraLastOldBridgeDepositTxNumber)
```

Comment:
why is it called deadline?

Reply:
I renamed it to `setEraLastOldBridgeDepositTime`, but if you have an even better name..

Reply:
time is good